sentry_sdk/integrations/aws_lambda.py

```diff
@@ -73,6 +131,14 @@ def _drain_queue():
 class AwsLambdaIntegration(Integration):
     identifier = "aws_lambda"
 
+    def __init__(self, check_timeout_error=False):
```

Comment:
`check_timeout_error` is a rather misleading name because it doesn't cause the integration to "check for a timeout error".

What it does is trigger a warning if you get "close enough" to the timeout.